dashboard/test/helpers/application_helper_test.rb

```diff
@@ -191,14 +191,18 @@ def request
     assert_equal '/sharing_drawing.png', meta_image_url(level: Artist.first)
     assert_equal '/studio_sharing_drawing.png', meta_image_url(level: Studio.first)
     assert_equal '/bounce_sharing_drawing.png', meta_image_url(level: Game.find_by_app('Bounce').levels.first)
-    assert_equal '/flappy_sharing_drawing.png', meta_image_url(level: Game.find_by_app('Flappy').levels.first)
+    level = create :level, game_id: 24
```

Comment:
We no longer have an levels in fixtures with Game 'Flappy' they are all 'CustomFlappy' now so I just switched to creating the level instead.

Reply:
This looks fragile, because it depends on a specific id. how about:
```
level = create :level, game_id: Game.custom_flappy.id
```
or just
```
level = create :level, game: Game.custom_flappy
```

Reply:
Updated